recipes/libsvtav1/all/test_package/conanfile.py

```diff
@@ -0,0 +1,26 @@
+import os
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import CMake, cmake_layout
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeToolchain", "CMakeDeps", "VirtualRunEnv"
```

Comment:
please add the v1 test package. it's required for the build to pass https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates/cmake_package/all/test_v1_package